Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -1,25 +1,28 @@
-import {Flex, Spacer} from "@chakra-ui/react";
+import {Flex, GridItem, SimpleGrid, Spacer} from "@chakra-ui/react";
 import PPC_CELL from "./PPC_CELL";
 import MPPT_CELL from "./MPPT_CELL";
 import OutputCurrent from "./OutputCurrent";
 import HeadingCell from "../Shared/HeadingCell";
 import RangeCell from "../Shared/RangeCell";
 import CONSTANTS from "../../data-constants.json";
+import {Grid} from "@chakra-ui/react";
+import {Box} from "@chakra-ui/react";
+
```

Comment:
Looks like SimpleGrid isn't being used, so that import can be removed. Also, the Grid (and Box, but I think Box can be removed entirely) import should be added on the same line as the other chakra ui imports.

Reply:
fixed

Reply:
@VIkranthMandadi, the `Grid` import should still be moved to the same line as `Flex`, `GridItem`, `SimpleGrid`, and `Spacer`. The `Box` import can be removed entirely because that component is not being used.

Reply:
fixed

Reply:
@VIkranthMandadi, imports still have to be updated/moved around.

Reply:
Awesome, thanks!